### DISCUSSION TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/data/work/temp2/sqan/refAnnotation_./output.genePred'

### DISCUSSION THREAD

**Discussion by handesome** [ConesaLab/SQANTI3#348] [Q&A] [answered]
Hi！Thank you for develop the package!!when I run the code `
sqanti3_qc.py /data/work/temp/gtf__talon.gtf /data/work/gencode.v46.primary_assembly.basic.annotation.gtf /data/work/GRCh38.primary_assembly.genome.fa \
--polyA_motif_list /opt/SQANTI3/data/polyA_motifs/mouse_and_human.polyA_motif.txt \
--coverage /data/work/pass2/pass2SJ.out.tab\
--force_id_ignore \
--skipORF \
--report pdf \
--dir /data/work/sqanti \
--output ./output`,the error says `
**** Parsing Reference Transcriptome....
mustOpen: Can't open /data/work/temp2/sqan/refAnnotation_./output.genePred to write: No such file or directory
Traceback (most recent call last):
  File "/opt/SQANTI3/sqanti3_qc.py", line 2627, in <module>
    main()
  File "/opt/SQANTI3/sqanti3_qc.py", line 2610, in main
    run(args)
  File "/opt/SQANTI3/sqanti3_qc.py", line 1896, in run
    refs_1exon_by_chr, refs_exons_by_chr, junctions_by_chr, junctions_by_gene, start_ends_by_gene = reference_parser(args, list(genome_dict.keys()))
                                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/SQANTI3/sqanti3_qc.py", line 676, in reference_parser
    for r in genePredReader(referenceFiles):
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/SQANTI3/sqanti3_qc.py", line 124, in __init__
    self.f = open(filename)
             ^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/data/work/temp2/sqan/refAnnotation_./output.genePred'`
so could anyone help me out?Thanks!!!!

**Comment by lkondratova** [comment, marked as answer]
This discussion is similar to #343: 
https://github.com/ConesaLab/SQANTI3/discussions/343#discussioncomment-11374597
